install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -220,19 +220,36 @@ metadata:
-  restuserkey: "password"  <5>
+  resturl: "http://127.0.0.1:8081" <1>
+  restuser: "admin" <2>
+  secretNamespace: "default" <3>
+  secretName: "heketi-secret" <4>
+  gidMin: "40000" <5>
+  gidMax: "50000" <6>
 
 ----
 
-<1> `glusterfs-cluster` is the endpoint name that includes GlusterFS trusted pool IP addresses. This parameter is mandatory. You also need to create a service for this endpoint so that the endpoint remains persistent. This service does not require a selector to tell {product-title} that the endpoints will be added manually. Please note that the `glusterfs` plug-in looks for the endpoint in the pod namespace. This means it is mandatory to create the endpoint and service in the Pod's namespace for a successful mount of gluster volumes in the pod.
-<2> Gluster REST service/Heketi service URL that provisions gluster volumes on demand. The general format should be `{http/https}://{IPaddress}:{Port}`. This is a mandatory parameter for GlusterFS dynamic provisioner. If the Heketi service is exposed as a routable service in the {product-title}, it will have a resolvable fully qualified domain name and Heketi service URL. For additional information and configuration, See link:https://access.redhat.com/documentation/en/red-hat-gluster-storage/3.1/single/container-native-storage-for-openshift-container-platform/[Container-Native Storage for OpenShift Container Platform].
-<3> A boolean value that indicates whether Gluster REST service authentication is enabled on the REST server. If this value is ‘true’, you must supply values for the ‘restuser’ and ‘restuserkey’ parameters.
-<4> Gluster REST service/Heketi user with access to create volumes in the Gluster Trusted Pool.
-<5> Gluster REST service/Heketi user's password used for authentication to the REST server. This parameter is deprecated in favor of `secretNamespace` and `secretName`.
+<1> Gluster REST service/Heketi service URL that provisions gluster volumes on demand. The general format should be `{http/https}://{IPaddress}:{Port}`. This is a mandatory parameter for GlusterFS dynamic provisioner. If the Heketi service is exposed as a routable service in the {product-title}, it will have a resolvable fully qualified domain name and Heketi service URL. For additional information and configuration, See link:https://access.redhat.com/documentation/en/red-hat-gluster-storage/3.1/single/container-native-storage-for-openshift-container-platform/[Container-Native Storage for OpenShift Container Platform].
+<2> Gluster REST service/Heketi user who has access to create volumes in the Gluster Trusted Pool.
+<3> Identification of Secret instance that containes user password to use when talking to Gluster REST service. These parameters are optional, empty password will be used when both `secretNamespace and secretName` are omitted. The provided secret must have type "kubernetes.io/glusterfs". When both restuserkey and `secretNamespace + secretName` is specified, the secret will be used.
```

Comment:
"When both restuserkey and \`secretNamespace + secretName\` is specified" : this does not make a lot of sense, since restuserkey is not mentioned any more. Does it still exist?

Apart from that s/is specified/are specified/
And appropriate \` quoting of "secretNamespace + secretName"